### PR TITLE
Fix monetary account joint retrieval bunq/sdk_java#45

### PR DESCRIPTION
**Pull request by OGKevin**
References #45

### CODE REVIEW

File: src/main/java/com/bunq/sdk/model/generated/endpoint/AttachmentConversationContent.java [hunk 1]
```diff
@@ -33,7 +33,6 @@
 
     return new BunqResponse<>(responseRaw.getBodyBytes(), responseRaw.getHeaders());
   }
-
```
Comment (patrickdw1991) [inline src/main/java/com/bunq/sdk/model/generated/endpoint/AttachmentConversationContent.java]: Why? :'(

Comment (patrickdw1991) [reply src/main/java/com/bunq/sdk/model/generated/endpoint/AttachmentConversationContent.java]: For all generated files

Comment (OGKevin) [reply src/main/java/com/bunq/sdk/model/generated/endpoint/AttachmentConversationContent.java]: Meh, lets see.

**Comment by OGKevin** [comment]
@patrickdw1991 regenerated.

**Comment by OGKevin** [comment]
@andrederoos

**Pull request merged**